src/main/java/net/minecraftforge/common/brewing/IBrewingRecipe.java

```diff
@@ -38,7 +40,7 @@ default boolean canCraftInDimensions(int p_43999_, int p_44000_)
         return ForgeMod.BREWING;
     }
 
-    Ingredient getReagent();
+    Predicate<ItemStack> getReagent();
```

Comment:
Nitpick: doesn't need a predicate, they can be methods themselves to be called. Seems weird to pass a predicate somewhere if it's never stored.